ecosystem/indexer-grpc/indexer-grpc-utils/src/cache_operator.rs

```diff
@@ -252,54 +366,57 @@ impl<T: redis::aio::ConnectionLike + Send + Clone> CacheOperator<T> {
         }
     }
 
-    // TODO: Remove this
-    pub async fn batch_get_encoded_proto_data(
+    pub async fn get_transactions_with_durations(
```

Comment:
this isn't used anywhere? if it's for refactoring we should split it out into another PR

Reply:
yes, but this collides with the function `batch_get_encoded_proto_data_x`; just rename here to avoid further confusion; we can address in refactoring.